buildSrc/settings.gradle.kts

```diff
@@ -1,6 +1,19 @@
 rootProject.name = "buildSrc"
 
+pluginManagement {
+   repositories {
+      mavenCentral()
```

Comment:
Do we need `mavenCentral()` for plugins?

Reply:
No, however, Maven Central is more strict and slightly more stable than GPP, so putting it first helps (slightly).